tokio/src/io/join.rs

```diff
@@ -0,0 +1,122 @@
+//! Join two values implementing `AsyncRead` and `AsyncWrite` into a single one.
+
+use crate::io::{AsyncRead, AsyncWrite, ReadBuf};
+
+use std::fmt;
+use std::io;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+cfg_io_util! {
```

Comment:
Please remove this `cfg_*` block. You're already wrapping the `mod` statement.

Reply:
Note that once you remove this block, you will get a bunch of failures from rustfmt. You will need to run rustfmt to format your code. See CONTRIBUTING.md for the appropriate command to use.